qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -549,45 +583,33 @@ def _get_sweep_data(self) -> np.ndarray:
         This function performs a sweep over the configured ranges.
         The result of the sweep is returned along with the sweep points
 
+        Args:
```

Comment:
where is this arg in the signature? (also there is a typo in the description)

Reply:
Removed in the last commit, had the change locally but forgot to commit